examples/basics.ipynb

```diff
@@ -0,0 +1,462 @@
+{
+ "cells": [
+  {
+   "source": [
+    "# Basics\n",
+    "\n",
+    "This notebook will introduce you to the basic data types and operations that are needed to use EOS."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "## Importing the notebook\n",
+    "\n",
+    "Before you can begin using EOS, you need to load it into your Python interpreter. We achieve this with the ``import`` keyword."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import eos"
+   ]
+  },
+  {
+   "source": [
+    "The import statement should go through without any error messages. If there are errors, they might related to the following issues:\n",
+    " - missing Python packages (try to run ``!pip3 install -U matplotlib numpy scipy``);\n",
+    " - you might have forgotten to tell the Python interpreter where you installed EOS to (should only happen if you have **not** installed EOS using ``pip``)"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "## (Almost) Everything can be a Parameter\n",
+    "\n",
+    "In EOS we make a lot of use of the ``eos.Parameter`` class. It provides access to one real-valued and scalar parameter, among a diverse and large set of parameters. You cannot create an object of type ``eos.Parameter`` yourself. Instead, you can create a new\n",
+    "set of Parameters, and extract your favourite parameter to get to know it."
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "parameters = eos.Parameters()"
+   ]
+  },
+  {
+   "source": [
+    "The variable ``parameters`` now contains all of your parameters. You can use ``display(parameters)`` to see the full list, which is rather lengthy. Let's have a look at something more manageable: the parameters for the muon mass:"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/plain": [
+       "<_eos.Parameter at 0x7f61ba9fff38>"
+      ],
+      "text/html": "\n    <table>\n        <tr>\n            <th>$$m_\\mu$$</th>\n            <td>(eos.Parameter)</td>\n        </tr>\n        <tr>\n            <th>current value</th>\n            <td><tt>0.105658</tt></td>\n        </tr>\n        <tr>\n            <th>default value</th>\n            <td><tt>0.105658</tt></td>\n        </tr>\n    </table>"
```

Comment:
Why is the muon mass latex rep rendered with two `\` ?

Reply:
Ah, that's because it needs to be escaped. Particular problem when converting from and to Python.
Python knows ``r'\mu'`` and uses only a single backslash. But ``'\\mu'`` needs two, to escape the backslash. Otherwise it would use the control character ``\m``, which is the carriage return.

At no point do we write ``\\`` in the code. In the YAML, using ``'...'`` ensures that. In Python, using ``r'...'`` "raw strings" ensures it. And in C++, using ``R"(...)"`` raw strings ensures that.